documentation/sphinx/source/developer-guide.rst

```diff
@@ -845,28 +845,72 @@ Metrics module
-  ('\xff\xff/metrics/data_distribution_stats/mako0457', '{"ShardBytes":297000}')
-  ('\xff\xff/metrics/data_distribution_stats/mako0524', '{"ShardBytes":264000}')
-  ('\xff\xff/metrics/data_distribution_stats/mako058', '{"ShardBytes":297000}')
-  ('\xff\xff/metrics/data_distribution_stats/mako064', '{"ShardBytes":297000}')
-  ('\xff\xff/metrics/data_distribution_stats/mako0718', '{"ShardBytes":264000}')
-  ('\xff\xff/metrics/data_distribution_stats/mako083', '{"ShardBytes":297000}')
-  ('\xff\xff/metrics/data_distribution_stats/mako0909', '{"ShardBytes":741000}')
+  ('\xff\xff/metrics/health/aggregate', '{"batch_limited":false,"tps_limit":483988.66315011407,"worst_storage_durability_lag":5000001,"worst_storage_queue":2036,"worst_log_queue":300}')
+  ('\xff\xff/metrics/health/log/e639a9ad0373367784cc550c615c469b', '{"log_queue":300}')
+  ('\xff\xff/metrics/health/storage/ab2ce4caf743c9c1ae57063629c6678a', '{"cpu_usage":2.398696781487125,"disk_usage":0.059995917598039405,"storage_durability_lag":5000001,"storage_queue":2036}')
+
+``\xff\xff/metrics/health/aggregate``
+
+Aggregate stats about cluster health. Reading this key alone is slightly cheaper than reading any of the per-process keys.
+
+============================ ======== ===============
+**Field**                    **Type** **Description**
+---------------------------- -------- ---------------
+batch_limited                boolean  Whether or not the cluster is limiting batch priority transactions
+tps_limit                    number   The rate at which normal priority transactions are allowed to start
+worst_storage_durability_lag number   See the description for storage_durability_lag
+worst_storage_queue          number   See the description for storage_queue
+worst_log_queue              number   See the description for log_queue
+============================ ======== ===============
+
+``\xff\xff/metrics/health/log/<id>``
+
+Stats about the health of a particular transaction log process
+
+========================= ======== ===============
+**Field**                 **Type** **Description**
+------------------------- -------- ---------------
+log_queue                 number   The number of bytes of mutations that need to be stored in memory on this transaction log process
+========================= ======== ===============
+
+``\xff\xff/metrics/health/storage/<id>``
+
+Stats about the health of a particular storage process
+
+========================== ======== ===============
+**Field**                  **Type** **Description**
+-------------------------- -------- ---------------
+cpu_usage                  number   The cpu percentage used by this storage process
+disk_usage                 number   The disk IO percentage used by this storage process
+storage_durability_lag     number   The difference between the newest version and the durable version on this storage process. On a lightly loaded cluster this will stay just above 5000000.
```

Comment:
replace `5,000,000` to "`MAX_READ_TRANSACTION_LIFE_VERSIONS`, which by default is 5,000,000 ", because we can configure the MVCC window larger than 5s.

Reply:
I added a footnote